Formula/payload-dumper-go.rb

```diff
@@ -0,0 +1,18 @@
+class PayloadDumperGo < Formula
+  desc "Android OTA payload dumper written in Go"
+  homepage "https://github.com/ssut/payload-dumper-go"
+  url "https://github.com/ssut/payload-dumper-go/archive/refs/tags/1.2.0.tar.gz"
+  sha256 "f0d654cba5d660c3ff8000cb510d73bc0a1eea4e725f2d79a1f16d973aae90c4"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "xz"
+
+  def install
+    system "go", "build", "-o", bin/"payload-dumper-go", "."
```

Comment:
```suggestion
    system "go", "build", *std_go_args
```

Reply:
Thanks, I applied and tested the change.